src/server/sass/build.js

```diff
@@ -76,8 +76,8 @@ export class Build {
       this.onSuccess(this);
     } catch(e) {
       this.onError(this, e);
```

Comment:
Not a fan of the way that the default `this.onError()` method is a noop that just swallows the error, and I think it would be more logical for `onSuccess` and `onError` to just rely on the promise returned from `Build#build()` (and in turn `buildAll()`) resolving/rejecting rather than implementing their own success/failure callbacks, but I realize that's kind of outside the scope of this fix.